handlers/graph.ts

```diff
@@ -1,9 +1,11 @@
+// TODO: Figure out how to integration with the Graph Service
```

Comment:
maybe create a ticket and remove this comment.

Reply:
Those are the todos of the tickets you are already working on right?

Reply:
Yes. my point we can remove these.

Reply:
All removed.